aws_pricing/README.md

```diff
@@ -0,0 +1,55 @@
+# Agent Check: AWS Pricing
+
+## Overview
+
+This check pulls in pricing information [published by AWS][1] to make it easier to measure cost of resource utilization in within Datadog.
+
+<div>Icons made by <a href="https://www.flaticon.com/authors/eucalyp" title="Eucalyp">Eucalyp</a> from <a href="https://www.flaticon.com/" 			    title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" 			    title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
```

Comment:
To be confirmed by the docs team but not sure we support html in markdown, I'll ask them to review.

Reply:
Plop, I'm not sure why this content is here. it's not related to the integration itself.. it should be removed i think.

Reply:
The I con I used for this integration is from the website above, they ask for attribution on use, so I was trying to abide by that license. I'll change it to markdown.

Reply:
Alternatively, if you guys know any designers, I'd be open to accepting a new icon for this integration :)